python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -333,6 +358,31 @@ def _get_num_gpus(
     return None
 
 
+def _get_num_tpus(
+    ray_start_params: Dict[str, str],
+    k8s_resource_limits: Dict[str, Any],
+    group_name: str,
+) -> Optional[int]:
+    """Get memory resource annotation from ray_start_params or k8s_resource_limits,
```

Comment:
What does this comment mean?

Reply:
Like with the CPU and GPU methods, I believe it's just referring to the resource requests provided to Ray either in `rayStartParams` or `spec: containers: resources`.

Reply:
Hmm ok. I know it's listed on CPU/GPU but this docstring doesn't tell me much, would it be better to just remove it altogether?

Reply:
I changed it to specify that we're checking the value provided for the TPU custom resource, either through ray_start_params or by specifying a `google.com/tpu` k8s_resource_limit.